contrib/babelfishpg_tds/src/backend/tds/tdsbulkload.c

```diff
@@ -993,6 +1004,7 @@ ReadBcpPlp(ParameterToken temp, StringInfo *message, TDSRequestBulkLoad request)
 	unsigned long lenCheck = 0;
 
 	CheckPlpMessageHasEnoughBytesToRead(message, sizeof(plpTok));
+	Assert(sizeof(plpTok)<=sizeof(plpTok));
```

Comment:
LHS = RHS, what is the point of the assert?

Reply:
Ok will remove it.